### ISSUE TITLE
CR: NCIDS - Remove Aria-Label and Add Alt-Text in Logo Index Link in Header

### ISSUE THREAD

**Issue by arcepaul** [NCIOCPL/ncids#423]
### Request Description
The logo index link in the header should be updated to be more accessible than it is currently.  Working with our PS accessibility resources, the image of the logo should have alt text instead of using aria-label.  All of the content, guidance and code examples should be updated to reflect the recommended approach to this element.

[W3C Source for requirement](https://www.w3.org/WAI/tutorials/images/functional/#example-1-image-used-alone-as-a-linked-logo)
[Webaim Source for requirement](https://webaim.org/techniques/alttext/#logos)

On the documentation site and storybook, the logo link image needs to have `alt-text` that is descriptive, such as "National Cancer Institute Homepage".

### Tasks

- [x] Create a Digital Platform ticket to update the header on prod to have the `alt-text` on the `<img src`. See Digital Platform ticket [# 4500](https://github.com/NCIOCPL/cgov-digital-platform/issues/4500)
- [x] Update the header on the doc site to have the `alt-text` on the `<img src` and remove the `aria-label`
- [x] Ensure accessibility information 
- [x] Update all header stories on Storybook to have appropriate `alt-text` on the `<img src` and remove the `aria-label`
- [x] Update the example site pages to have the appropriate `alt-text` on the `<img src` and remove the `aria-label`

**Comment by meglake** [comment]
@arcepaul, as time permits please add your content/guidance to be included in the NCIDS so that this ticket can be assigned to be completed soon after MVP 1 launch.
